NewHorizons/Builder/Props/DetailBuilder.cs

```diff
@@ -316,5 +318,32 @@ private static void FixComponent(Component component, GameObject planetGO, strin
                 }
             });
         }
+
+        private static void AddPhysics(GameObject prop, Sector sector)
+        {
+            var primaryBody = prop.GetComponentInParent<OWRigidbody>();
+
+            var rb = prop.AddComponent<Rigidbody>();
+            var owrb = prop.AddComponent<OWRigidbody>();
+            var kine = prop.AddComponent<KinematicRigidbody>();
+            rb.isKinematic = true;
+            var offsetApplier = prop.AddComponent<CenterOfTheUniverseOffsetApplier>();
+            offsetApplier.Init(owrb);
+            owrb._simulateInSector = sector;
+            owrb._kinematicSimulation = true;
+            owrb._kinematicRigidbody = kine;
+            owrb._offsetApplier = offsetApplier;
+
+            prop.AddComponent<InitialMotion>().SetPrimaryBody(primaryBody);
+            prop.AddComponent<MatchInitialMotion>().SetBodyToMatch(primaryBody);
+
+            var detector = new GameObject("Detector");
+            detector.transform.parent = prop.transform;
+            detector.transform.localPosition = Vector3.zero;
+
+            var shape = detector.AddComponent<SphereShape>();
```

Comment:
This probably needs its radius set to fit the mesh bounds inside. It may also be good to make this part of the config so the addon can decide what shape and size the shape should be

Reply:
> This is awesome, it’ll be a fun feature to have

If only it actually worked lol